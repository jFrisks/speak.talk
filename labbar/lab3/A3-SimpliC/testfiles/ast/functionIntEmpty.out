Program
  List
    FunctionDecl ID="lucas"
      List
      Block
        List
