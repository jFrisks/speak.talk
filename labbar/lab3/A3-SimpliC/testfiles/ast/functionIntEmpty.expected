Program
  List
    FunctionDecl
      IdDecl ID="lucas"
      List
      Block
        List